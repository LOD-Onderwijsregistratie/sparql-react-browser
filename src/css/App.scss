path.link {
  fill: none;
  stroke: #666;
  stroke-width: 1.5px;
}

text {
  text-anchor: middle;
  font-family: "Helvetica Neue", Helvetica, sans-serif;
  fill: #666;
  font-size: 16px;
}

.active{
   // color: green;
   // background-color: red;
   border-bottom: 3px solid blue;
   font-weight: bold;
}

.nav-link{
   
}

.navbar-toggler{
   border-bottom: none;
}
  
.nav-link:hover {
  border-bottom: 3px solid ligh;
  //font-weight: bold;
}

.navbar{
   padding-left: 5%;
   padding-right: 5%;
   padding-top: 0px;
   padding-bottom: 0px;
   background-color: white;
   box-shadow: 0 0 15px ;   
}

.dropdown-item.active{
   background-color: white;
   color: black;
}

.PageContent{
   padding-top: 50px;
   min-height: 100vh;
}

.Footer{
   background-color: #222222;
   color: white;
   text-align: center;
}

.textarea{
   min-width: 100%;

   min-height: 300px;
   background-color: red;
}

.ellipse {
  fill: cyan;
  stroke: #333;
  stroke-width: 1.5px;
}

.uri {
  fill: cyan;
}

.literal {
  fill: red;
}

.faded {
  opacity: 0.3;
}



.ellipse_highlight {
  fill: yellow;
  stroke: blue;
  stroke-width: 2px;
}

#forcegraph {
  height: 800px;
}

div.tooltip {	
    position: absolute;			
    text-align: center;			
    		
  }