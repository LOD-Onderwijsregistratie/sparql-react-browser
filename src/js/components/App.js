import React from "react";
import DataSourceDropdown from "./DataSourceDropdown";

const App = () => (
  <div>
    <h2>Select DataSourceDropdown</h2>
    <div className="content">
      <DataSourceDropdown/>
      </div>
  </div>
);

export default App;
