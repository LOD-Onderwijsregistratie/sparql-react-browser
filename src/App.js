import React from 'react';

//Router
import { ConnectedRouter } from 'connected-react-router'
import { history } from './redux/store/middleware'

//bootstrap stylesheet
import 'bootstrap/dist/css/bootstrap.min.css';

//custom stylesheet
import './css/App.scss';

//devtools
import DevTools from './containers/DevTools.jsx'

import { Provider } from 'react-redux'

import configureStore from './redux/store/index'

//Bootrap components
import Container from 'react-bootstrap/Container'

// Layout components
import Header from './components/Header'
import Footer from './components/Footer'
import PageContent from './components/PageContent';
import ConnectedLoading from './containers/ConnectedLoading'
import ConnectedError from './containers/ConnectedError'

//init redux store
const store = configureStore()
const initialState = store.getState()

console.log('initializing store: ', store.getState())

const unsubscribe = store.subscribe(() => {
  //console.log('state changed: ', store.getState())
})




const App = () => (
  <Provider store={store}>
    <div className="App">
      <ConnectedRouter history={history}>
        <Header />
        <Container>
          <PageContent />
          <ConnectedLoading />
          <ConnectedError />
        </Container>

        <Footer />
      </ConnectedRouter>
      {/* exclude devtools in production */}
      {(process.env.NODE_ENV !== 'production') && <DevTools />}
    </div>
  </Provider>
);

export default App;

