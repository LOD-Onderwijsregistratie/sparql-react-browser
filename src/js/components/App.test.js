import { render } from '@testing-library/react';
import App from './App';
import React from 'react';

jest.mock('./DataSourceDropdown', () => 'p');
jest.mock('./SearchComponent', () => 'p');

describe('App component', () => {
  it('should render component', ()=> {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
