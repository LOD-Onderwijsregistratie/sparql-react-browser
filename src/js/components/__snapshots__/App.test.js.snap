// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render component 1`] = `
<DocumentFragment>
  <div>
    <h2>
      Select DataSourceDropdown
    </h2>
    <div
      class="content"
    >
      <p />
    </div>
  </div>
</DocumentFragment>
`;
